tests/darwin/item_sorter_test.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from darwin.item_sorter import ItemSorter, SortDirection
+
+
+def describe_item_sorter():
+    def describe_parse():
+        def test_works_when_sort_is_complete_and_valid():
```

Comment:
I would suggest getting rid of these `test_` prefixes, now we can 🙂 
```suggestion
        def works_when_sort_is_complete_and_valid():
```

Reply:
Done